src/util/error-handler.ts

```diff
@@ -74,4 +80,23 @@ export const defaultErrorHandler: ErrorHandler = async (ctx: ChatInputCommandInt
     } else {
         await ctx.reply(reply);
     }
+
+    // ----- !COMMAND REPLY -----
+
+    // ----- ERROR LOG -----
+
+    const user: User = ctx.user;
+    const command: ApplicationCommand = ctx.command;
+
+    const report: string = "```\n" +
+                           "Command Error\n\n" +
+                           `Server: ${ctx.guild.name}\n` +
+                           `User: ${user.id} (${user.username})\n` +
+                           `Command: ${command.name}\n` +
+                           `Options: ${command.options.toString()}\n\n` +
+                           // Error name, and error code if it's a Discord API error
+                           `${err.name}${err instanceof DiscordAPIError ? `: ${err.code} (${err.message})` : ""}\n` +
+                           "```";
+    
+    await getAdminLogsChannel().send(report);
```

Comment:
This has been added to the command error handler to make it log to the error log in the admin server